app/playbooks/config.d/cluster/config.py

```diff
@@ -25,7 +25,8 @@ def __init__(self, path, footer, rtr_interfaces):
             ])
         self.cluster = ParameterCollection('cluster', 'Cluster Configuration', [
             PasswordParameter('ADMIN_PASSWORD', 'Adminstrator Password'),
-            PasswordParameter('PULL_SECRET', 'Pull Secret')
+            PasswordParameter('PULL_SECRET', 'Pull Secret'),
+            Parameter('FIPS_MODE', 'Fips Mode')
```

Comment:
This parameter should be of BooleanParameter type. Also Fips should be FIPS as it is an acronym. 

```
BooleanParameter('FIPS_MODE', 'FIPS Mode', 'False')
```
Note that False is of string type. This will set the default value to False.